static/js/pages/applications/ApplicationDashboardPage_test.js

```diff
@@ -0,0 +1,64 @@
+/* global SETTINGS: false */
+import { assert } from "chai"
+
+import ApplicationDashboardPage from "./ApplicationDashboardPage"
+import IntegrationTestHelper from "../../util/integration_test_helper"
+import { makeApplication } from "../../factories/application"
+import { makeUser } from "../../factories/user"
+
+describe("ApplicationDashboardPage", () => {
+  let helper,
+    fakeUser,
+    renderPage,
+    fakeApplications = []
+
+  beforeEach(() => {
+    helper = new IntegrationTestHelper()
+    fakeApplications = [makeApplication(), makeApplication()]
+    fakeUser = makeUser()
+
+    helper.handleRequestStub.withArgs("/api/applications/").returns({
+      status: 200,
+      body:   fakeApplications
+    })
+
+    renderPage = helper.configureReduxQueryRenderer(
+      ApplicationDashboardPage,
+      {},
+      {
+        entities: {
+          currentUser: fakeUser
+        }
+      }
+    )
+  })
+
+  afterEach(() => {
+    helper.cleanup()
+  })
+
+  it("renders a card for each application that a user has", async () => {
```

Comment:
Can we add a test or logic to this test to assert some of the text in a card, for example that the profile name and bootcamp title exist, and that the application status text lookup produced the right text?